cmd/manager/main.go

```diff
@@ -50,9 +64,21 @@ func main() {
 	log.Info(fmt.Sprintf("KUDO Version: %s", fmt.Sprintf("%#v", version.Get())))
 
 	// create new controller-runtime manager
-	log.Info("setting up manager")
+	syncPeriod, err := parseSyncPeriod()
+	if err != nil {
+		log.Error(err, "unable to parse manager sync period variable")
+		os.Exit(1)
+	}
+
+	if syncPeriod != nil {
+		log.Info(fmt.Sprintf("setting up manager, sync-period is %v", syncPeriod))
```

Comment:
```suggestion
		log.Info("setting up manager, sync-period is %v", syncPeriod)
```

Reply:
Per `log.Info` usage reference, (Info(msg string, keysAndValues ...interface{})), looks like it doesn't support string formatting. So I think we shall keep this line.

Reply:
Oh, you're right! Thanks for noticing, as we use multiple loggers as `log` with different feature set in the same file. We'll need to improve on this. In this context `log.Info(fmt.Sprintf` is the right approach, will resolve these comments.

Reply:
You're right, thanks for noticing!